src/python/WMCore/WMSpec/StdSpecs/TaskChain.py

```diff
@@ -252,13 +266,15 @@ def __call__(self, workloadName, arguments):
                 if isGenerator(arguments):
                     # generate mc events
                     self.workload.setWorkQueueSplitPolicy("MonteCarlo", taskConf['SplittingAlgo'],
-                                                          taskConf['SplittingArguments'])
+                                                          taskConf['SplittingArguments'],
+                                                          BlowupFactor=blowupFactor)
                     self.workload.setEndPolicy("SingleShot")
                     self.setupGeneratorTask(task, taskConf)
                 else:
                     # process an existing dataset
                     self.workload.setWorkQueueSplitPolicy("Block", taskConf['SplittingAlgo'],
-                                                          taskConf['SplittingArguments'])
+                                                          taskConf['SplittingArguments'],
+                                                          BlowupFactor=blowupFactor)
```

Comment:
I was going to comment on the case of this argument, but I guess it's passed through to MonteCarlo.py?

Upper case is definitely wrong, but the mistake is not yours.